cola/app.py

```diff
@@ -178,6 +178,8 @@ def __init__(self, context, argv, locale=None,
 
         self.context = context
         self._install_hidpi_config()
+        # Make icons sharp in HiDPI screen
+        self._app.setAttribute(Qt.AA_UseHighDpiPixmaps, True)
```

Comment:
This makes sense to be in the ColaQApplication constructor instead.

We still support PyQt4, so please use check `hasattr(Qt, 'AA_UseHighDpiPixmaps')` before using it here.

This code probably doesn't work as written, and should be hitting a traceback -- self._app hasn't been assigned yet -- it's getting assigned on the line below.